rules/S6371/cfamily/metadata.json

```
@@ -0,0 +1,20 @@
+{
+  "title": "Coroutine states should be released",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
```

I think fixing that would require a lot more time.
```suggestion
    "constantCost": "1h"
```